eZ/Publish/Core/FieldType/RichText/Resources/stylesheets/ezxml/docbook/core.xsl

```diff
@@ -188,19 +192,33 @@
 
   <xsl:template match="anchor">
     <xsl:element name="anchor" namespace="http://docbook.org/ns/docbook">
-      <xsl:attribute name="xml:id">
-        <xsl:value-of select="@name"/>
-      </xsl:attribute>
+      <xsl:if test="count(key('ids', @name)) = 1">
```

Comment:
You could also use `<xsl:choose>` (https://msdn.microsoft.com/pl-pl/library/ms256169(v=vs.120).aspx) here and below, but two `<xsl:if>` statments are also fine for me 😉

Reply:
yeah. in this case it doesn't make much difference.. but since `<xsl:choose>` is the way to do if-else in xsl, I'll change it

Reply:
fixed